lib/modules/platform/local/scm.ts

```diff
@@ -0,0 +1,45 @@
+import { glob } from 'glob';
+import type { CommitFilesConfig, CommitSha } from '../../../util/git/types';
+import type { PlatformScm } from '../types';
+
+let fileList: string[] | undefined;
```

Comment:
This is ok for now because we only allow one repo per run

Reply:
you could reset in init repo, or throw there if someone tries multiple repos

Reply:
Let's limit local to one repo